android/app/src/main/python/mmrelay/cli.py

```
@@ -0,0 +1,1614 @@
+        bool: True if the platform supports E2EE and all required dependencies can be imported;
+              False if running on an unsupported platform (Windows) or if any dependency is missing.
+    """
+    if sys.platform == WINDOWS_PLATFORM:
+        print("❌ Error: E2EE is not supported on Windows")
+        print("   Reason: python-olm library requires native C libraries")
+        print("   Solution: Use Linux or macOS for E2EE support")
+        return False
+
+    # Check if E2EE dependencies are available
+    try:
+        import olm  # noqa: F401
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)